lib/baw-workers/harvest/single_file.rb

```diff
@@ -292,6 +292,24 @@ def rename_harvested_file(file_path, storage_target_paths)
             @logger.info(@class_name) {
               "Source file #{file_path} was successfully renamed to #{renamed_source_file}."
             }
+
+            # if this is a wav file, and a .wac file exists, rename the .wac file as well
+            file_ext = File.extname(file_path)
+            wac_file = file_path[0..-4]+'wac'
+            if file_ext == '.wav' && File.exists?(wac_file)
+              renamed_wac_file = wac_file + '.completed'
+              File.rename(wac_file, renamed_wac_file)
+              if File.exists?(renamed_wac_file)
+                @logger.info(@class_name) {
+                  "Source .wac file #{wac_file} was successfully renamed to #{renamed_wac_file}."
+                }
+              else
+                @logger.warn(@class_name) {
+                  "Source .wac file #{wac_file} was not renamed."
+                }
+              end
+            end
```

Comment:
So this block itself is fine, but the implication is disturbing: if we harvest files in WAac then we'll use triplicate storage until `harvester_to_do` is cleaned up. Given #50 & #36 this pattern will become more frequent as well.

I think we should keep source file in `harvest_to_do`, and delete the converted file out of `harvester_to_do` after it is copied to `original_audio`. Thoughts?